### DISCUSSION TITLE
Struggling with Linux FRRounting and nProbe

### DISCUSSION THREAD

**Discussion by marcantoinegodde** [akvorado/akvorado#154] [Q&A] [answered]
Hello,

Our edge BGP router is a router based on Debian 11 and FRRouting. I discovered Akvorado at the last FRnog.

I'm currently trying to deploy Akvorado on this router. Therefore, I installed nProbe on the machine and configured it to export NetFlow v9 on a virtual machine via UDP (port 2055), I added the option `-S=1:1:1` to define the sampling rate.

I receive flows on the VM but at the beginning, I had this error `sampling rate missing` I fixed with `default-sampling-rate: 1`

Now, I experience `input and output interfaces missing`. I didn't configure SNMP yet since I have no idea on which OID I should monitor.

Any help would be appreciated.

**Comment by vincentbernat** [comment]
I think just installing NetSNMP on your Linux box should work. Akvorado is looking for `sysName.0` and interface alias and descriptions in the IF-MIB.

**Comment by marcantoinegodde** [comment]
Ok. Si basically, since I found no IP settings to configure SNMP in Akvorado, I guess that Akvorado polls SNMP metrics directly from the IP which sent the NetFlow data. Right ?

**Reply by vincentbernat** [reply]
Yes.

**Reply by vincentbernat** [reply]
This will be possible to specify IP addresses to use for each exporter in the next version. See af58717da682.

**Reply by marcantoinegodde** [reply]
Ah oui… la fonctionnalité est ajoutée instant ! 😂 Bravo.

For English readers: the feature is added instantly ! Congrats.

**Comment by marcantoinegodde** [comment, marked as answer]
I finally succeeded in configuring nProbe to add the required fields in the flows such as the Interface SNMP Index.
I also succeeded in polling SNMP with a dirty firewall exception rule since the last version of Akvorado is not released yet.

Now, sadly, I struggle with an easy part... I downloaded both GeoLite2-ASN and GeoLite2-Country into `/usr/share/GeoIP/`, I changed `optional` to `false`. I face this error: `akvorado-inlet_1            | Error: unable to start component: cannot open geo database: open /usr/share/GeoIP/GeoLite2-Country.mmdb: no such file or directory`. I checked permissions just in case, the problem seems not to be there.

**Reply by vincentbernat** [reply]
The inlet is running in a container and does not have access to `/usr/share/GeoIP` on the host. Have a look at `docker-compose.yml` to make it work (or alter it to use `/usr/share/GeoIP` from host as a volume).
